pkg/cli/cvd.go

```diff
@@ -238,15 +242,62 @@ func listCVDs(c *cobra.Command, flags *ListCVDsFlags, opts *subCommandOpts) erro
 					fmt.Errorf("lists cvds for host %q failed: %w", name, err))
 				return
 			}
-			for _, cvd := range hostCVDs {
-				printCVDs(c.OutOrStdout(), buildBaseURL(flags.CVDRemoteFlags), name, []*hoapi.CVD{cvd})
-			}
-		}(host)
+			out := &bytes.Buffer{}
+			printCVDs(out, buildBaseURL(flags.CVDRemoteFlags), name, hostCVDs)
+			b, _ := io.ReadAll(out)
```

Comment:
this seems more complicated than necessary: why not have a function that returns a string and have the caller print it however it wants?

Reply:
true, this was removed after addressing the other comment